README.md

```diff
@@ -58,6 +58,12 @@ Go 1.21+
   performance, and includes advanced features such as serializable ACID
   transactions, write batches, compression, and more.
 
+- **[PebbleDB](https://github.com/cockroachdb/pebble) [experimental]:** Pebble
```

Comment:
Maybe add here the note existing in ForceSync that people using the SDK might have a problem with this.

Reply:
created https://github.com/cometbft/cometbft-db/issues/126